static/bidder-info/vidazoo.yaml

```diff
@@ -0,0 +1,18 @@
+endpoint: "https://prebidsrvr.cootlogix.com/openrtb/"
+maintainer:
+  email: "dev@vidazoo.com"
+gvlVendorID: 744
+endpointCompression: gzip
+capabilities:
+  app:
+    mediaTypes:
+      - banner
+      - video
+  site:
+    mediaTypes:
+      - banner
+      - video
+userSync:
+  iframe:
+    url: https://sync.cootlogix.com/api/user/html/pbs_sync?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&redirect={{.RedirectURL}}
+    userMacro: "${userId}"
```

Comment:
above url is not redirecting or replacing `userMacro` with user id

```
https://sync.cootlogix.com/api/user/html/pbs_sync?gdpr=&gdpr_consent=&us_privacy=&redirect=https%3A%2F%2Fprebid.adnxs.com%2Fpbs%2Fv1%2Fsetuid%3Fbidder%3Dcootlogix%26gdpr%3D%26gdpr_consent%3D%26f%3Di%26uid%3D%22%24%7BuserId%7D%22
```

Reply:
@onkarvhanumante 
It will return HTML code with the redirect as a pixel.
Did you send it from the EU? maybe it was blocked by gdpr

Reply:
I tried this in APAC region. AFAIK, This is not the expected behaviour from iframe user sync url. You would still need to return the redirect URL with proper userID.

Reply:
@gargcreation1992 
Fixed - will now return HTML with the redirect and useID.
Can you please check?

Reply:
<img width="1588" alt="image" src="https://github.com/prebid/prebid-server/assets/24757781/76b7d85a-e152-4972-8751-36af6c1cf4b7">

user id was not appended for 
`https://sync.cootlogix.com/api/user/html/pbs_sync?gdpr=&gdpr_consent=&us_privacy=&redirect=https%3A%2F%2Fprebid.adnxs.com%2Fpbs%2Fv1%2Fsetuid%3Fbidder%3Dcootlogix%26gdpr%3D%26gdpr_consent%3D%26f%3Di%26uid%3D%22%24%7BuserId%7D%22`

Reply:
@onkarvhanumante Updated the vidazoo.yaml - the userMacro should be without quotes.

Reply:
`https://sync.cootlogix.com/api/user/html/pbs_sync?gdpr=&gdpr_consent=&us_privacy=&redirect=https%3A%2F%2Fprebid.adnxs.com%2Fpbs%2Fv1%2Fsetuid%3Fbidder%3Dcootlogix%26gdpr%3D%26gdpr_consent%3D%26f%3Di%26uid%3D%24%7BuserId%7D`


iframe url is working